optaplanner-wb-screens/optaplanner-wb-solver-editor/optaplanner-wb-solver-editor-api/src/main/java/org/optaplanner/workbench/screens/solver/service/SolverEditorService.java

```diff
@@ -39,6 +41,7 @@
         SupportsCreate<SolverConfigModel>,
         SupportsRead<SolverConfigModel>,
         SupportsUpdate<SolverConfigModel>,
+        SupportsSaveAndRename<SolverConfigModel, Metadata>,
         SupportsDelete,
         SupportsCopy,
         SupportsRename {
```

Comment:
You can probably remove ```SupportsRename``` as you're now extending ```SupportsSaveAndRename``` that extends ```SupportsRename``` (yeah, yeah, same old comment.. I guess this may not be the last time!)